evap/student/tests/test_views.py

```diff
@@ -204,3 +204,15 @@ def test_midterm_evaluation_warning(self):
 
         page = self.get_assert_200(self.url, user=self.voting_user1.username)
         self.assertIn(evaluation_warning, page)
+
+    @override_settings(SMALL_COURSE_SIZE=5)
+    def test_participants_warning_shown(self):
+        participants_warning = "Only a small number of people can take part in this evaluation."
+        page = self.get_assert_200(self.url, user=self.voting_user1.username)
+        self.assertIn(participants_warning, page)
+
+    @override_settings(SMALL_COURSE_SIZE=2)
+    def test_participants_warning_not_shown(self):
+        participants_warning = "Only a small number of people can take part in this evaluation."
+        page = self.get_assert_200(self.url, user=self.voting_user1.username)
+        self.assertNotIn(participants_warning, page)
```

Comment:
please put both the changes to the tests as well as the new tests into their respective commits, a commit with a bunch of changes and new tests related to two separate commits is harder to review.